assembly/src/main/resources/features.xml

```diff
@@ -96,6 +97,27 @@
         <bundle>mvn:org.apache.karaf.cellar/org.apache.karaf.cellar.cloud/${project.version}</bundle>
     </feature>
 
+    <feature name="cellar-kubernetes" description="Kubernetes support in clusters" version="${project.version}" resolver="(obr)">
+        <configfile finalname="/etc/org.apache.karaf.cellar.kubernetes.cfg">
+            mvn:org.apache.karaf.cellar/apache-karaf-cellar/${project.version}/cfg/kubernetes
+        </configfile>
+        <feature version="${project.version}">cellar-core</feature>
+        <feature>cxf-jaxrs</feature>
```

Comment:
In case you're wondering, I've opened https://github.com/fabric8io/fabric8/issues/3346